Meetme.ProfileService/Meetme.ProfileService.DAL/Entities/Profile.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Meetme.ProfileService.DAL.Entities.Enums;
+using Meetme.ProfileService.DAL.Entities.Interfaces;
+
+namespace Meetme.ProfileService.DAL.Entities;
+
+public class Profile : BaseEntity, ITimestamped
```

Comment:
If you inherit BaseEntity, ITimestamped, may be there is a reason to do like:

BaseEntity: ITimestamped

?